pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,202 @@
+# Running Buildkite Agent on Google Cloud Platform
+
+The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). For fine control over long–lived agents, you might like to run the agent using individual VM instances on Google Compute Engine. Or run Docker–based builds using a scalable cluster of agents on the Google Container Engine.
+
+<%= toc %>
+
+## Running the agent on Google Compute Engine
+
+To run the agent on your own [Google Compute Engine](https://cloud.google.com/compute) instance use whichever installer matches your instance type. For example:
+
+Launch an instance using the latest Ubuntu LTS image [through the console](https://console.cloud.google.com/compute/instancesAdd#preconfigured-image-ubuntu-1604-xenial-v20170202):
+
+<%= image "create-an-instance.png", size: "2200x1216", alt: "Screenshot of creating a Google Compute Engine instance running Ubuntu 16.04 LTS via the Google Cloud Console" %>
+
+Connect via SSH:
+
+<%= image "ssh-to-instance.png", size: "2048x1220", alt: "Screenshot of connecting to a Google Compute Engine instance via the Google Cloud Console" %>
+
+Now follow the [setup instructions for Ubuntu](/docs/agent/ubuntu).
```

Comment:
The previous steps have all started with verbs, so this one should too. If you just remove the 'now' it works well. Would be good to add in 'Buildkite agent' or something of the like before 'setup instructions' too.